reanalysis/hail_filter_and_label.py

```diff
@@ -845,11 +843,15 @@ def main(
     parser.add_argument(
         '--out_vcf', type=str, required=True, help='VCF path to export results'
     )
+    parser.add_argument(
+        '--tmp_path', type=str, required=True, help='Temp path for checkpoints'
```

Comment:
I'm not sure how much you've integrated the recent config changes, but should this be a parameter or use the `hail_batch.output_path` function (with a `tmp` category) in `cpg-utils` instead?

Reply:
I am using and passing it through as an argument, but it does make more sense to just do that here!